### ISSUE TITLE
Update to VTK9

### ISSUE THREAD

**Issue by Punzo** [Punzo/SlicerAstroApp#3]
SlicerAstro extension has been update to VTK 9 https://github.com/Punzo/SlicerAstro/commit/7d93ff20c01a39b9ee083493c989eb8c65596594

and this SlicerAstroApp as well https://github.com/Punzo/SlicerAstroApp/commit/d1546413d13714a220724cd51072a3a0c17d491b

@vdhulst probably it will be good to redo binaries, but it would be better to wait a little bit to fix last few issues (e.g.: https://github.com/Slicer/Slicer/issues/5220).

**Comment by vdhulst** [comment]
@Punzo: yes, I agree it is better to wait with building a new distribution until the other issues are resolved, unless you want your changes tested under MacOS. 
Thanks for producing this update.

**Comment by Punzo** [comment]
> unless you want your changes tested under MacOS.

no let's wait, I just wanted to ping you to give an update (: 

> Thanks for producing this update.

no problem!

**Comment by vdhulst** [comment]
@Punzo: got a new laptop with Catalina MacOS 10.15.7, and the binary distribution we offer at present installed and ran just fine.

**Comment by Punzo** [comment]
> @Punzo: got a new laptop with Catalina MacOS 10.15.7, and the binary distribution we offer at present installed and ran just fine.

good 😄  ! when issues #5223, #5220 and #5250 in Slicer will be fixed, we can redo the binaries using Qt5.15.1 and VTK9 (in the SlicerAstro repos, everything is already ready for the updgrade).
